at-hoangtran/Week3/Day2/Exercise3.rb

```diff
@@ -0,0 +1,25 @@
+# Hold data
+lecturers = [
+  {lecturer: 'Paul Barry', subject:  ['Programming', 'Networking', 'Security', 'Open Source', 'Frameworks']},
+  {lecturer: 'Chris Meudec', subject:  ['Testing', 'Safety Systems', 'Formal Systems', 'Programming Languages']},
+  {lecturer: 'Nigel Whyte', subject:  ['Graphics', 'Imaging', 'Programming', 'Sign Language', 'Trees']},
+  {lecturer: 'Austin Kinsella', subject:  ['Networks', 'WANs', 'Programming', 'Macintosh', 'Digital Photography']},
+  {lecturer: 'Gerry Moloney', subject:  ['Placement', 'Employment', 'Emerging Systems', 'Web Development']},
+]
+
+# Show list areas of interest of the first and third lecturers
+puts 'List areas of interest of the first:'
+puts lecturers[0][:subject]
+puts 'List areas of interest of the third:'
+puts lecturers[2][:subject]
+
+# Remove the last lecturer
+puts 'Remove the last lecturer'
+puts lecturers.delete_at(lecturers.length-1)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/569f251521ee030730363aec7c55804496be311f/at-hoangtran/Week3/Day2/Exercise3.rb#L18"></span>Layout/SpaceAroundOperators: Surrounding space missing for operator <code>-</code>. (https://github.com/rubocop-hq/ruby-style-guide#spaces-operators)</td>
    </tr>
  </tbody>
</table>